examples/tests/sdk_tests/service/stateful.test.w

```diff
@@ -0,0 +1,89 @@
+bring cloud;
```

Comment:
@ainvoner let me know what you think about this

Reply:
looks good.

Reply:
yes